dns/ddclient/src/opnsense/mvc/app/models/OPNsense/DynDNS/DynDNS.xml

```diff
@@ -14,6 +14,10 @@
                 <default>0</default>
                 <Required>Y</Required>
             </verbose>
+            <allowipv6 type="BooleanField">
+                <default>0</default>
+                <Required>Y</Required>
```

Comment:
This requires a model version bump in the same model file (see above)

Reply:
Like so? Or do we need a migration as well?

Reply:
that's ok, defaults are migrated automatically.